src/Angor/Client/wwwroot/lib/js-base64/base64.mjs

```diff
@@ -0,0 +1,294 @@
+/**
```

Comment:
@sondreb any chance you can review this part of the code? especially around password generator and the way I imported the base64 js files